packages/blue-sdk/test/Market.test.ts

```diff
@@ -0,0 +1,148 @@
+import {
+  AdaptiveCurveIrm__factory,
+  BlueOracle__factory,
+  ERC20__factory,
+  MorphoBlue__factory,
+} from "ethers-types";
+
+import { SignerWithAddress } from "@nomicfoundation/hardhat-ethers/signers";
+
+import { setUp } from "@morpho-org/morpho-test";
+import { Time } from "@morpho-org/morpho-ts";
+import {
+  latest,
+  setNextBlockTimestamp,
+} from "@nomicfoundation/hardhat-network-helpers/dist/src/helpers/time";
+import { ethers } from "hardhat";
+import { deal } from "hardhat-deal";
+import { ChainId, Market, MarketConfig, MathLib, addresses } from "../src";
+
+describe("Market", () => {
+  let signer: SignerWithAddress;
+
+  setUp(async () => {
+    signer = (await ethers.getSigners())[0]!;
+  });
+
+  it("should borrow borrowable assets", async () => {
```

Comment:
Maybe you should check that borrowing a bit more than `maxBorrowableAssets` reverts?